sources/packages/forms/src/form-definitions/institutionprofile.json

```diff
@@ -23,7 +25,27 @@
         "json": ""
       },
       "customConditional": "",
-      "logic": [],
+      "logic": [
+        {
+          "name": "Client type Not AEST",
+          "trigger": {
+            "type": "javascript",
+            "javascript": "result = (data.clientType !== 'aest')"

```

Comment:
I believe that this is where the API value is being consumed. Does it need an API concern to return it.
IMO it could be provided by the Vue application.